cli/cmd/common/utils.go

```diff
@@ -1,20 +1,81 @@
 package common
 
 import (
+	"embed"
 	"fmt"
 	"os"
 	"strings"
 
 	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chart/loader"
 	helmCLI "helm.sh/helm/v3/pkg/cli"
 	"k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
 const (
 	DefaultReleaseName      = "consul"
 	DefaultReleaseNamespace = "consul"
+	chartFileName           = "Chart.yaml"
+	valuesFileName          = "values.yaml"
+	templatesDirName        = "templates"
+	TopLevelChartDirName    = "consul"
 )
 
+// ReadChartFiles reads the chart files from the embedded FS, and loads their contents into []*loader.BufferedFile. This
```

Comment:
Can we say "file system" instead of FS? Just for clarity.

```suggestion
// ReadChartFiles reads the chart files from the embedded file system, and loads their contents into []*loader.BufferedFile. This
```

Reply:
Overall, this is a great comment! Makes things super clear!